web/modules/custom/dpl_loans/src/Controller/DplLoansController.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Drupal\dpl_loans\Controller;
+
+use Drupal\Core\Block\BlockManagerInterface;
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\Core\Render\RendererInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException;
+
+/**
+ * Render loan list react app.
+ */
+class DplLoansController extends ControllerBase {
+
+  /**
+   * DplLoansController constructor.
+   *
+   * @param \Drupal\Core\Block\BlockManagerInterface $blockManager
+   *   Drupal block manager.
+   * @param \Drupal\Core\Render\RendererInterface $renderer
+   *   Drupal renderer service.
+   */
+  public function __construct(
+    private BlockManagerInterface $blockManager,
+    private RendererInterface $renderer
+  ) {
+  }
+
+  /**
+   * {@inheritdoc}
+   *
+   * @param \Symfony\Component\DependencyInjection\ContainerInterface $container
+   *   The Drupal service container.
+   *
+   * @return static
+   */
+  public static function create(ContainerInterface $container) {
+    return new static(
+      $container->get('plugin.manager.block'),
+      $container->get('renderer'),
+    );
+  }
+
+  /**
+   * Display page with the loan list.
+   *
+   * @return mixed[]
+   *   Render array.
+   *
+   * @throws \Drupal\Component\Plugin\Exception\PluginException
+   */
+  public function list(): array {
+    $config = [];
+
+    /** @var \Drupal\dpl_loans\Plugin\Block\LoanListBlock $plugin_block */
+    $plugin_block = $this->blockManager->createInstance('dpl_loans_list_block', $config);
+
+    // Some blocks might implement access check.
```

Comment:
This is really minor. But why do you write "blocks" in plural when we know there is only one block (`dpl_loans_list_block`) that is being used. Which is used (`dpl_loans_list_block::build()`) in order to render the list in yhis method. If I have understood it correctly :)